samples/bluetooth/central_iso/src/main.c

```diff
@@ -18,20 +18,53 @@
 #include <zephyr/bluetooth/iso.h>
 #include <zephyr/settings/settings.h>
 #include <zephyr/sys/byteorder.h>
+#include <zephyr/sys/util.h>
+
+/* Currently only nRF5 hardware is supported by this sample. */
+#include "bt_clock_hal_nrf5.h"
+
+#define LATENCY_MS      20U /* 20.0 ms, FT=2 with CIS reliability policy, otherwise w/o effect */
+#define SDU_INTERVAL_US (10U * USEC_PER_MSEC) /* 10.0 ms */
+#define NUM_RETRIES     1U                    /* SDU tx retries, influences ISO interval length */
+
+/* We define an application, HCI transport and controller specific guard time
+ * which needs to protect against a race between SDU preparation and transmission
+ * under worst-case conditions including:
+ * - time between scheduling SDU preparation work and its actual execution from
+ *   the work queue
+ * - SDU preparation time
+ * - HCI transport latency
+ * - internal controller proccessing latency
+ */
+#define SDU_PREPARATION_GUARD_TIME_US 3000U
 
 static void start_scan(void);
 
 static struct bt_conn *default_conn;
-static struct k_work_delayable iso_send_work;
 static struct bt_iso_chan iso_chan;
-static uint16_t seq_num;
-static uint16_t latency_ms = 10U; /* 10ms */
-static uint32_t interval_us = 10U * USEC_PER_MSEC; /* 10 ms */
-NET_BUF_POOL_FIXED_DEFINE(tx_pool, 1, BT_ISO_SDU_BUF_SIZE(CONFIG_BT_ISO_TX_MTU),
+
+static struct bt_iso_tx_info last_iso_tx_sync_info;
+static inline bool is_synchronized(void)
```

Comment:
```suggestion
static bool is_synchronized(void)
```

`inline`s are rarely an improvement, and should generally be left to the compiler